src/xref2/compile.ml

```diff
@@ -64,7 +64,7 @@ and content env id =
   | Module m ->
       let sg = Type_of.signature env m in
       Module (signature env (id :> Id.Signature.t) sg)
-  | Pack _ -> failwith "Unhandled content"
+  | Pack p -> Pack p
```

Comment:
How hard would it be to actually compile packs ?

Reply:
I've not even looked into it to be honest - not many packages use packing right now, but the one I have installed was causing my tests to fail!